bcmi/src/app/footer/footer.component.html

```diff
@@ -0,0 +1,80 @@
+<footer class="app-footer">
+	<div class="container">
+		<div class="row" *ngIf="footerData">
+			<div class="col-md-4">
+				<section>
+					<h4>{{footerData.About_title}}</h4>
+					<div class="mb-0" [innerHTML]="footerData.About_description"></div>
+				</section>
+			</div>
+			<div class="col-md-4">
+				<section>
+					<h4>Navigate</h4>
+					<div *ngFor="let nav of navigation">
+						<h5>{{nav.attributes.Heading}}</h5>
+						<ul>
+							<li *ngFor="let link of nav.attributes.pages.data">
+							<a [routerLink]="link.attributes.route">{{link.attributes.Title}}</a>
+							</li>
+						</ul>
+					</div>
+				</section>
+			</div>
+			<div class="col-md-4">
+				<section class="last">
+					<h4>{{footerData.Connect_title}}</h4>
+					<div [innerHTML]="footerData.Connect_description"></div>
+					<a class="btn slide-r-btn inverted submit" href="/contact" [routerLink]="['/contact']"><span>Submit Feedback</span><i class="material-icons">arrow_forward</i></a>
+				</section>
+			</div>
+		</div>
+		<div class="footer-admin clearfix">
+			<ul class="gov-links">
+				<li><a href="#">Home</a></li>
+				<li><a href="http://www2.gov.bc.ca/gov/content/home/copyright" target="_blank" rel="noopener">Copyright</a></li>
+				<li><a href="http://www2.gov.bc.ca/gov/content/home/disclaimer" target="_blank" rel="noopener">Disclaimer</a></li>
+				<li><a href="http://www2.gov.bc.ca/gov/content/home/privacy" target="_blank" rel="noopener">Privacy</a></li>
+				<li><a href="http://www2.gov.bc.ca/gov/content/home/accessibility" target="_blank" rel="noopener">Accessibility</a></li>
+			</ul>
+			<ul class="connect-links">
+				<li>
+					<div class="fb-share-button"
+						data-href="https://mines.nrs.gov.bc.ca"
```

Comment:
Is there a reason why a non-relative link is used here?

Reply:
data-href is a way of passing the href to be shared to facebook, see https://developers.facebook.com/docs/plugins/share-button/. However not sure it's actually being used here